### PR TITLE
Fix HTML5 notify domain verification instructions

### PR DESCRIPTION
**Pull request by DavidDeSloovere**
Didn't say you need to restart.

Can't get the internal link working

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by DavidDeSloovere** [comment]
It's still messy, I mean the order in which the different blocks are listed is very confusing, but at least one major item of information was added: the reboot.

**Comment by frenck** [comment]
Judging by the merge conflict, this one has been solved already.
@DavidDeSloovere What do you think?

**Comment by DavidDeSloovere** [comment]
@frenck The page that's currently online (https://home-assistant.io/components/notify.html5/) does still not contain the most important part for the verification process: restart is needed after placing a file in the /www dir. I believe this PR is still needed.

File: source/_components/notify.html5.markdown [hunk 1]
```diff
@@ -33,30 +33,32 @@ Configuration variables:
...
+1. Make sure you can access your Home Assistant installation from outside your network over https ([see docs](https://home-assistant.io/docs/configuration/remote/)).
+2. Create a new project at [https://console.cloud.google.com/home/dashboard](https://console.cloud.google.com/home/dashboard).
+3. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain via Google Webmaster Central / Search Console - [hass.io instructions](#verify-your-domain-with-hassio).
+4. With the domain verified, go to [https://console.firebase.google.com](https://console.firebase.google.com), select import Google project, and select the project you created.
+5. Then, click the cogwheel on top left and select "Project settings".
+6. Select 'Cloud Messaging' tab, listed beneath Project Credentials will be your 152 character 'Server Key' and 12 digit ID 'Sender ID' you need for configuring this component.
 
 #### {% linkable_title Verify your domain with Hass.io %}
```
Comment (DubhAd) [inline source/_components/notify.html5.markdown]: Can you update the title from Hass.io to Home Assistant please

File: source/_components/notify.html5.markdown [hunk 2]
```diff
@@ -33,30 +33,32 @@ Configuration variables:
...
-1. Set `https://yourdomain/local/` as URL.
-2. For verifying your domain you need to download a file in step 2.
-3. Create a directory named "www" in you Hass.io configuration directory.
-4. Place the file (something like this: google*.html) in the "www" directory.
-5. Verify the domain.
-6. Proceed with step 3.
+Follow these steps to verify domain ownership with Google Webmaster Central / Search Console:
+1. Enter your domain and add **'/local'** at the end, ie. https://whitehouse.duckdns.org:8123/local
+2. Select HTML file verification and download the google*.html file.
+2. Create a directory named "www" in your Hass.io configuration directory (/config share from Samba add-on).
```
Comment (DubhAd) [inline source/_components/notify.html5.markdown]: Similarly, Hass.io -> Home Assistant

File: source/_components/notify.html5.markdown [hunk 3]
```diff
@@ -33,30 +33,32 @@ Configuration variables:
...
-2. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain.
-3. After that, go to [https://console.firebase.google.com](https://console.firebase.google.com) and select import Google project, select the project you created.
-4. Then, click the cogwheel on top left and select "Project settings".
-5. Select 'Cloud Messaging' tab, listed beneath Project Credentials will be your 152 character 'Server Key' and 12 digit ID 'Sender ID'.
+1. Make sure you can access your Home Assistant installation from outside your network over https ([see docs](https://home-assistant.io/docs/configuration/remote/)).
+2. Create a new project at [https://console.cloud.google.com/home/dashboard](https://console.cloud.google.com/home/dashboard).
+3. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain via Google Webmaster Central / Search Console - [hass.io instructions](#verify-your-domain-with-hassio).
```
Comment (DubhAd) [inline source/_components/notify.html5.markdown]: Hass.io -> Home Assistant please

**Comment by DavidDeSloovere** [comment]
@DubhAd Made it unambiguous (it that even the correct word 😄 )

File: source/_components/notify.html5.markdown [hunk 4]
```diff
@@ -33,30 +33,32 @@ Configuration variables:
...
+2. Create a new project at [https://console.cloud.google.com/home/dashboard](https://console.cloud.google.com/home/dashboard).
+3. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain via Google Webmaster Central / Search Console - [instructions](#verify-your-domain).
+4. With the domain verified, go to [https://console.firebase.google.com](https://console.firebase.google.com), select import Google project, and select the project you created.
+5. Then, click the cogwheel on top left and select "Project settings".
+6. Select 'Cloud Messaging' tab, listed beneath Project Credentials will be your 152 character 'Server Key' and 12 digit ID 'Sender ID' you need for configuring this component.
+
+#### {% linkable_title Verify your domain %}
+
+Follow these steps to verify domain ownership with Google Webmaster Central / Search Console:
+1. Enter your domain and add **'/local'** at the end, ie. https://whitehouse.duckdns.org:8123/local
```
Comment (DubhAd) [inline source/_components/notify.html5.markdown]: Only one thing I think now - `whitehouse.duckdns.org` is a real domain, use `duckdns.example.org`?

**Comment by c727** [comment]
Fake news: `+4. RESTART Home Assistant - this is important!`

**Comment by DavidDeSloovere** [comment]
@c727 Could you be more specific? I'm 100% sure it only worked for me *after* restarting HA.

**Comment by c727** [comment]
You don't have to restart HA after you have made changes in www

**Comment by DubhAd** [comment]
Given the number of people who've had to restart HA to get files in `www/` picked up, which it may not be officially required, it seems to be functionally required in many cases.

**Pull request merged**